scripts/pack-debug.sh

```
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+script_dir="$( cd "$( dirname "$0"  )" && pwd  )"
+project_dir=$(dirname "$script_dir")
+systype=$(uname -s)
+build_type="debug"
+
+. $script_dir/fns.sh --source-only
+
+cd $project_dir;
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)